index.html

```diff
@@ -4,12 +4,84 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;700&display=swap" rel="stylesheet">
+  <link rel="icon" type="image/png" href="./favicon.png">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css">
   <link rel="stylesheet" href="./css/app.css">
-  <title>Testing linters</title>
+  <title>Apple page mockup - microverse project</title>
 </head>
 
 <body>
-  <h1>Hello world!</h1>
+  <div class="grid-container">
+
+    <header>
+
+      <nav>
+        <ul class="nav">
+          <li><a href=""><i class="fab fa-apple"></i></a></li>
+          <li><a href="">Store</a></li>
+          <li><a href="">Mac</a></li>
+          <li><a href="">ipod</a></li>
+          <li><a href="">iphone</a></li>
+          <li><a href="">iPad</a></li>
+          <li><a href="">iTunes</a></li>
+          <li><a href="">Support</a></li>
+          <li>
+            <label for="search"><input type="text" id="search"><i class="fas fa-search"></i></label>
+          </li>
+        </ul>
+
+      </nav>
+      <div class="hero">
+        <a href="#">
+          <h1 class="hero-title">What will your verse be?</h1>
+          <p class="hero-subtitle">Everyone has something to share. See how people are using iPad in their unique ways
+            to
+            add to the world’s story.</p>
+        </a>
+        <a href="#">
+          <p class="hero-link">Watch the "Your Verse" film and explore the stories
+            <i class="fas fa-play-circle"></i>
+          </p>
+        </a>
+
+      </div>
+    </header>
+
+
+    <main class="main-container">
+      <div class="item"><a href="#"><img src="./img/promo_iphone5s.jpg" alt="iPhone 5s"></a></div>
+      <div class="item"><a href="#"><img src="./img/promo_iphone5c.jpg" alt="iPhone 5c"></a></div>
+      <div class="item"><a href="#"><img src="./img/promo_30_years.jpg" alt="30 years"></a></div>
+      <div class="item"><a href="#"><img src="./img/promo_supplier_responsibility.jpg" alt="Responsibility"></a>
+      </div>
+    </main>
```

Comment:
## Main container

Your images should be behind each other, they're not accomplishing as the original one

![image](https://user-images.githubusercontent.com/24902525/87672005-e35ef000-c737-11ea-9a38-22bdf94d0537.png)


- [ ] Use `position:absolute;` and `z-index-1;`